src/rules/function-color-channel/index.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+
+const valueParser = require("postcss-value-parser");
+const { utils } = require("stylelint");
+const declarationValueIndex = require("../../utils/declarationValueIndex");
+const namespace = require("../../utils/namespace");
+const ruleUrl = require("../../utils/ruleUrl");
+
+const ruleName = namespace("function-color-channel");
+
+const messages = utils.ruleMessages(ruleName, {
+  rejected: "Expected the color.channel function to be used"
```

Comment:
Small detail, but should we mention the function name here? 

E.g.

```js
 rejected: (functionName) => `Expected the color.channel function to be used instead of ${functionName}`
```

Reply:
We don't mention the function name in `function-color-relative`'s message as well, so I had copied that. Also, it should be clear from the reported range already, which covers only the function name.

Reply:
Renamed both in [`d25f7da`](https://github.com/stylelint-scss/stylelint-scss/pull/1060/commits/d25f7da41f08346a23c7b1fe548a9501a68ad2eb).